charts/hono/example/add_example_data_device_registry.sh

```diff
@@ -35,7 +35,7 @@ add_tenant(){
   HTTP_REQUEST_BODY=$2
 
   echo "Adding tenant [$TENANT_ID]"
-  HTTP_RESPONSE=$(curl -w "%{http_code}" \
+  HTTP_RESPONSE=$(curl -o /dev/null -sw "%{http_code}" \
```

Comment:
@sophokles73 the code looked like this before. You have changed it. But I can't see how this would work.

Reply:
Not sure what you mean.
```sh
> curl -o /dev/null -sw "%{http_code}" -X GET "http://hono.eclipseprojects.io/credentials/DEFAULT_TENANT/4711"
404
```
Doesn't this work for you?

Reply:
Yes. This works as I expect it, i.e. only returning the status code. What does not work is the statement without the `-o` and `-s` parameters.

Reply:
That's how it looks to me.